proto/prysm/v1alpha1/attestation/aggregation/attestations/attestations.go

```diff
@@ -27,11 +27,11 @@ var ErrInvalidAttestationCount = errors.New("invalid number of attestations")
 // Aggregation occurs in-place i.e. contents of input array will be modified. Should you need to
 // preserve input attestations, clone them before aggregating:
 //
-//   clonedAtts := make([]*ethpb.Attestation, len(atts))
-//   for i, a := range atts {
-//       clonedAtts[i] = stateTrie.CopyAttestation(a)
-//   }
-//   aggregatedAtts, err := attaggregation.Aggregate(clonedAtts)
+//	clonedAtts := make([]*ethpb.Attestation, len(atts))
```

Comment:
why does your PR always do this?

Reply:
have no clue, I just run update-go-pbs

Reply:
That's weird. Our `./hack/update-go-pbs.sh` don't do this. Hopefully we don't mess the next person up